src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements Areacount,Figuredrawing {
+    private int radius;
+
+    public Circle(int radius,String color) {
+        this.radius = radius;
+        setColor(color);

```

Comment:
let's use the constructor of Figure class to initialize the color field
the same with other classes